.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Don't `git add .` or you will end up committing files by mistake to your codebase.

Commit each file one by one and on purpose, using the file name.